up42/catalog.py

```diff
@@ -110,6 +110,31 @@ def get_collections(self) -> Union[Dict, List]:
         collections = [c for c in json_response["data"] if c["type"] == self.type]
         return collections
 
+    def estimate_order(self, order_parameters: Union[dict, None], **kwargs) -> dict:
+        """
+        Estimate the cost of an order.
+
+        Args:
+            order_parameters: A dictionary like {dataProduct: ..., "params": {"id": ..., "aoi": ...}}
+
+        Returns:
+            dict: representation of a JSON estimation response with summary, results, and errors..
+
+        Warning "Deprecated order parameters"
```

Comment:
It comes from previous warnings placed in the SDK before. I am not sure what was the format they used before. The team that started the SDK is not anymore in the company, maybe we can agree a new format.

Reply:
Please notice that this function was not created in this PR, it was moved from the child class to the parent class. Mostly, because now estimation is also possible to be used in the Tasking Class. Therefore, we need to move it to make it available in tasking.

All the comments for this function are part of the previous implementation. We can tackle it here, or we can address this in a refactor ticket.